tailwind.config.ts

```diff
@@ -0,0 +1,149 @@
+import type { Config } from "tailwindcss";
+
+const config: Config = {
+	content: [
+		"./pages/**/*.{js,ts,jsx,tsx,mdx}",
+		"./components/**/*.{js,ts,jsx,tsx,mdx}",
+		"./app/**/*.{js,ts,jsx,tsx,mdx}",
+	],
+	important: true,
+	darkMode: "class",
+	theme: {
+		extend: {
+			backgroundImage: {
+				"gradient-radial": "radial-gradient(var(--tw-gradient-stops))",
+				"gradient-conic":
+					"conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))",
+			},
+			fontFamily: {
+				sans: ["var(--font-inter)"],
+			},
+			theme: {
```

Comment:
Did you export this theme from Figma?

Reply:
Exported as json from Figma. Already exists in flood-frontend and developer-portal.